### DISCUSSION TITLE
Two installations Thorium v2.3.0 and Thorium v3.0.0

### DISCUSSION THREAD

**Discussion by Pacdoc-Oz** [edrlab/thorium-reader#2684] [Q&A]
Using Windows 10 Prof on Asus laptop 64bit 
I originally installed Thorium because Internet Archive allows me to download loans as a reading disabled person in the LCP epub or pdf formats and I can read them with large print settings. 

How can I handle this duplication.  
I have desktop icon which opens Thorium v2 with one book
I have an icon on the Start bar which opens v3 with all the books except the one opened by v2
Should I uninstall both versions and start again and if so, will I lose the books in the library of both

Could it be that I must find all the EDRL files and delete them all by hand?

**Comment by danielweck** [comment]
Hello, the publication storage area (i.e. folder on the filesystem) for Thorium official releases is different than the one for test / alpha builds. I wonder if this is why you are seeing different bookshelf contents.
On Windows, there is possibly the added confusion introduced by the Microsoft app store which may use a different installation folder, and perhaps a different data folder too?
How did you install Thorium? (Windows store or via the installer downloaded from the official releases?)

**Comment by Pacdoc-Oz** [comment]
Hi Daniel - only ever use official sources reached by URL of official
releases and manually rather than clicking update
